api/MediaSettingsRange.json

```diff
@@ -0,0 +1,196 @@
+{
+  "api": {
+    "MediaSettingsRange": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaSettingsRange",
+        "support": {
+          "webview_android": {
+            "version_added": "59"
+          },
+          "chrome": {
+            "version_added": "59"
+          },
+          "chrome_android": {
+            "version_added": "59"
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": null
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": "46"
+          },
+          "opera_android": {
+            "version_added": "46"
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": false,
```

Comment:
Looks to be [`true`](https://w3c.github.io/mediacapture-image/#mediasettingsrange-section)